internal/ui/etui_event_handlers.go

```diff
@@ -28,7 +29,7 @@ func handleAppUpdateAvailable(_ context.Context, fr *frame.Frame, event partybus
 		return err
 	}
 
-	message := color.Magenta.Sprintf("New version of %s is available: %s", internal.ApplicationName, newVersion)
+	message := color.Magenta.Sprintf("New version of %s is available: %s (current version is %s)", internal.ApplicationName, newVersion, version.FromBuild().Version)
```

Comment:
(see comment in other instance of this message. Also, should we consider factoring out this message into a shared value?)

Reply:
my take, it's better as-is (not being a shared constant); the ETUI and logging are different-enough of concerns to not need a single source of truth for this message (and they have their own rules for how something is displayed). For instances, in logging the sentence should start with a lower case but in the ETUI it should be capitalized.